.github/workflows/linux.yml

```diff
@@ -12,7 +12,9 @@ jobs:
 
   test_on_alpine:
     runs-on: ubuntu-latest
-    container: alpine
+    container:
+      image: alpine
+      options: --security-opt seccomp=unconfined
```

Comment:
Add an `# comment` explainig why we need this, otherwise this will be a very puzzling line for someone unfamiliar with the context :rofl: